src/components/LeafletMap.js

```diff
@@ -250,4 +253,25 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+
+  findAutocompleteLocation = async (value) => {
+    try {
+      let location = await getAutocompleteMapLocation(value.trim());
+      let resource = location.resourceSets[0].resources[0];
+      let center = resource.point.coordinates;
+      const markerIcon = L.icon({ iconUrl: L.Icon.Default });
+      this.map.setView(center, 5);
+      const marker = new L.marker(center, { icon: markerIcon });
+      //marker.onAdd = ()=>setTimeout( ()=>this.map.setZoomAround(center,5),500);
```

Comment:
Might be good to remove this comment to keep things clean.

Reply:
removed comment